src/main/java/cn/nukkit/network/protocol/PlayerListPacket.java

```diff
@@ -43,6 +43,11 @@ public void encode() {
             }
         }
 
+        if (type == TYPE_ADD) {
+            for (Entry entry : this.entries) { // Biggest wtf
+                this.putBoolean(entry.skin.isTrusted());
+            }
+        }
```

Comment:
This seems like a way to achieve backwards compatibility, probably done before the protocol was bumped. Unneeded to assume incompetence - lots of people, lots of devs.